packages/renderer/src/lib/cronjob/CronJobActions.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+import { createEventDispatcher } from 'svelte';
+
+import { withConfirmation } from '/@/lib/dialogs/messagebox-utils';
+
+import ListItemButtonIcon from '../ui/ListItemButtonIcon.svelte';
+import type { CronJobUI } from './CronJobUI';
+
+export let cronjob: CronJobUI;
+export let detailed = false;
+
+const dispatch = createEventDispatcher<{ update: CronJobUI }>();
```

Comment:
issue: createEventDispatcher is deprecated so I would not introduce new code using it
https://svelte.dev/docs/svelte/v5-migration-guide#Event-changes-Component-events